shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -950,4 +957,28 @@ public void onFlutterUiNoLongerDisplayed() {
     // no-op
   }
 
+  /**
+   * Registers all plugins that an app lists in its pubspec.yaml.
```

Comment:
minor nit: plugins that support the android platform

Reply:
I think that would be redundant to point out here. This is already a Javadoc comment in the Android embedding specifically and not a generic part of the engine's C++ source code, so I don't think it's worth pointing out that this logic is Android-only. If you feel strongly about it I can update it though.

Reply:
Not at all. 🙂Although, if someone wants to understand why a plugin isn't added to the `GeneratedPluginRegistrant`, the comment might be useful. 

The current definition of supporting the Android platform is a plugin that contains `android/build.gradle`.

https://api.flutter.dev/javadoc/io/flutter/app/FlutterActivity.html.